test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -195,10 +195,34 @@ public void execute_list_showsAllPersons() throws Exception {
         helper.addToAddressBook(addressBook, false, true);
 
         assertCommandBehavior("list",
-                              Command.getMessageForPersonListShownSummary(expectedList),
-                              expectedAB,
-                              true,
-                              expectedList);
+                Command.getMessageForPersonListShownSummary(expectedList),
+                expectedAB,
+                true,
+                expectedList);
+    }
+
+    @Test
+    public void execute_list_showAllPerson() throws Exception {
+        //prepare expectations
+        TestDataHelper helper = new TestDataHelper();
+        Person person1 = helper.generatePersonWithName("Airy");
+        Person person2 = helper.generatePersonWithName("Bily");
+        Person person3 = helper.generatePersonWithName("Cily");
+
+        List<Person> expectedList = helper.generatePersonList(person1,person2,person3);
+        List<Person> unexpectedList = helper.generatePersonList(person2,person3,person1);
+
+        AddressBook expectedAB = helper.generateAddressBook(unexpectedList);
+
+        //prepare Addressbook state
+        helper.addToAddressBook(addressBook,unexpectedList);
+
+        assertCommandBehavior("sort",
```

Comment:
This is a good "Positive" test. You can also consider writing tests to cover the negative scenarios.
While for this exercise there is no need to update any more tests, in your project you should definitely write tests covering both positive and negative cases.